src/boxy/textures.nim

```diff
@@ -91,10 +90,9 @@ proc newTexture*(image: Image): Texture =
   result.componentType = GL_UNSIGNED_BYTE
   result.format = image.getFormat()
   result.internalFormat = GL_RGBA8
-  result.genMipmap = false
-  result.minFilter = minLinear
+  result.genMipmap = true
+  result.minFilter = minLinearMipmapLinear
```

Comment:
For the future; could decouple minFilter from mipFilter. gl is the only one coupling these two

Reply:
I will add that as a task.